admin/class-admin.php

```diff
@@ -133,8 +133,17 @@ function register_settings_page() {
 
 		// Add main page.
 		/* translators: %s: number of notifications */
-		$counter_screen_reader_text = sprintf( _n( '%s notification', '%s notifications', $notification_count ), number_format_i18n( $notification_count ) );
-		$counter = sprintf( '<span class="update-plugins count-%1$d"><span class="plugin-count" aria-hidden="true">%1$d</span><span class="screen-reader-text">%2$s</span></span>', $notification_count, $counter_screen_reader_text );
+		$counter_screen_reader_text = esc_attr( sprintf( _n( '%s notification', '%s notifications', $notification_count ), number_format_i18n( $notification_count ) ) );
```

Comment:
Minor improvement, just for consistency: prefer to wrap the `esc_attr` in the `sprintf` building the HTML output.